src/_includes/layouts/homepage.html

```diff
@@ -6,35 +6,43 @@
     <link rel="stylesheet" href="css/reset.css" />
     <link rel="stylesheet" href="css/styles.css" />
     <link rel="stylesheet" href="/css/navbar.css" />
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@900&family=Source+Code+Pro:ital,wght@0,400;0,600;1,500;1,600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/css/homepage.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@900&family=Source+Code+Pro:ital,wght@0,400;0,600;1,500;1,600&display=swap"
+      rel="stylesheet"
+    />
     <title>Lunch.dev Community Calendar</title>
   </head>
   <body data-typeface="system-ui">
-	{% include partials/header.html %}
-    <main data-layout="centered-single-column">
-      <h1>Lunch.dev Community Calendar</h1>
+    {% include partials/header.html %}
 
-      <h2 id="upcoming-events-section">Upcoming Events</h2>
-      <ul aria-labelledby="upcoming-events-section">
+    <main data-layout="centered-single-column">
+      <h2>Upcoming Events</h2>
+      <section id="upcoming-events-section" class="events-section">
         {% for event in collections.upcomingEvents %} {{ event | log }}
-        <li>
-          <a href="{{ event.url }}"> {{ event.data.title }} </a>
+        <article aria-labelledby="{{ event.data.title | slug }}" class="event-item">
+          <h3 id="{{ event.data.title | slug }}">{{ event.data.title }}</h3>
           {% if event.data.type %}
-          <span>{{ event.data.type }}</span>
+          <div>{{ event.data.type }}</div>
           {% endif %}
-        </li>
+          <a href="{{ event.url }}" aria-label="View Details for '{{ event.data.title }}'">View Details &#8250;</a>
+        </article>
         {% endfor %}
-      </ul>
+      </section>
 
-      <h2 id="past-events-section">Past Events</h2>
-      <ul aria-labelledby="past-events-section">
+      <h2>Past Events</h2>
+      <section id="past-events-section" class="events-section">
         {% for event in collections.pastEvents %} {{ event | log }}
-        <li>
-          <a href="{{ event.url }}"> {{ event.data.title }} </a>
-        </li>
+        <article aria-labelledby="{{ event.data.title | slug }}" class="event-item">
+          <h3 id="{{ event.data.title | slug }}">{{ event.data.title }}</h3>
+          {% if event.data.type %}
+          <div>{{ event.data.type }}</div>
+          {% endif %}
+          <a href="{{ event.url }}" aria-label="View Details for '{{ event.data.title }}'">View Details &#8250;</a>
```

Comment:
@BenDMyers I wrapped the event title in single quotes in the aria-label here, what do you think? unnecessary?

Reply:
Probably unnecessary, I think. Depending on the screenreader user's verbosity settings, it might even be a bit of extra clutter, since verbose screenreaders might announce something like "View Details for apostrophe Eleventy at 11 apostrophe, link." I'd suggest removing the apostrophes.